src/yesql/generate.clj

```diff
@@ -72,21 +72,27 @@
 ;; case, we only ever use one group, so we'll unpack the
 ;; single-element list with `first`.
 (defn execute-handler
-  [db sql-and-params call-options]
-  (first (jdbc/execute! db sql-and-params)))
+  [db sql params call-options]
+  (first (jdbc/execute! db (rewrite-query-for-jdbc sql params))))
 
 (defn insert-handler
-  [db [statement & params] call-options]
-  (jdbc/db-do-prepared-return-keys db statement params))
+  [db sql params call-options]
+  (if (vector? params)
+    (let [table-string-position 0
+          table-position        2
+          table-keyword         (keyword ((split (sql table-string-position) #" ") table-position))]
+      (apply jdbc/insert! db table-keyword params))
+    (let [[rewritten-sql & rewritten-params] (rewrite-query-for-jdbc sql params)]
+      (jdbc/db-do-prepared-return-keys db rewritten-sql rewritten-params))))
```

Comment:
keeping this instead of replacing with insert! for backwards compatibility purposes.